demo/config.yaml

```diff
@@ -21,3 +21,9 @@ lcr-modules:
         inputs:
             sample_bam: "data/{sample_id}.bam"
             sample_bai: "data/{sample_id}.bam.bai"
+
+    strelka:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+            candidate_indel: "results/manta-2.0/99-outputs/vcf/{seq_type}--{genome_build}/candidateSmallIndels/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf"
```

Comment:
We should try to think of a way to simplify these inter-module connections for users. Any ideas?

Reply:
hmm I'm not sure, we could symlink it instead but that would still require the user to specify where the file is. 
And maybe we could add a default location `config["lcr-modules"]["manta"]["dirs"]["outputs"]/vcf/{seq_type}--{genome_build}/candidateSmallIndels/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf"`. 

Not sure if this makes things simpler